packages/cucumber-browserstack-example/step-definitions/steps.js

```diff
@@ -18,5 +18,5 @@ Then(/^the Google search form exists$/, async () => {
 });
 
 Then(/^the DuckDuckGo search form exists$/, async () => {
-  await client.assert.visible('#search_form_input_homepage');
+  await client.assert.visible('.js-search-input.search__input--adv');
```

Comment:
Can we use an id instead? Class can be changed any time.

Reply:
Makes sense. Updated it.